src/NServiceBus.Core/Receiving/ReceiveComponent.cs

```diff
@@ -112,33 +112,33 @@ static TransportTransactionMode GetRequiredTransactionMode(Settings settings, Tr
 
             receiveComponent.BindQueues(transportConfiguration.QueueBindings);
```

Comment:
there would also be no need to bind the receiving queues I think

Reply:
Agred